x/auth/vesting/client/cli/tx.go

```diff
@@ -79,3 +88,175 @@ timestamp.`,
+Coins may not be transferred out of the account if they are locked or unvested, but may be staked.
+Staking rewards are subject to a proportional vesting encumbrance.
+
+A periods file is a JSON object describing a sequence of unlocking or vesting events,
+with a start time and an array of coins strings and durations relative to the start or previous event.`,
+		Example: `Sample period file contents:
+{
+  "start_time": 1625204910,
+  "periods": [
+    {
+      "coins": "10test",
+      "length": 2592000 // 30 days
+    },
+    {
+      "coins": "10test",
+      "length": 2592000 // 30 days
+    }
+  ]
+}
+`,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			toAddr, err := sdk.AccAddressFromBech32(args[0])
+			if err != nil {
+				return err
+			}
+
+			lockupFile, _ := cmd.Flags().GetString(FlagLockup)
+			vestingFile, _ := cmd.Flags().GetString(FlagVesting)
+			if lockupFile == "" && vestingFile == "" {
+				return fmt.Errorf("must specify at least one of %s or %s", FlagLockup, FlagVesting)
+			}
+
+			var (
+				lockupStart, vestingStart     int64
+				lockupPeriods, vestingPeriods []types.Period
+			)
+			if lockupFile != "" {
+				lockupStart, lockupPeriods, err = readScheduleFile(lockupFile)
+				if err != nil {
+					return err
+				}
+			}
+			if vestingFile != "" {
+				vestingStart, vestingPeriods, err = readScheduleFile(vestingFile)
+				if err != nil {
+					return err
+				}
+			}
+
+			commonStart, _ := types.AlignSchedules(lockupStart, vestingStart, lockupPeriods, vestingPeriods)
+
+			merge, _ := cmd.Flags().GetBool(FlagMerge)
+
+			msg := types.NewMsgCreateClawbackVestingAccount(clientCtx.GetFromAddress(), toAddr, commonStart, lockupPeriods, vestingPeriods, merge)
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	cmd.Flags().Bool(FlagMerge, false, "Merge new amount and schedule with existing ClawbackVestingAccount, if any")
+	cmd.Flags().String(FlagLockup, "", "path to file containing unlocking periods")
+	cmd.Flags().String(FlagVesting, "", "path to file containing vesting periods")
+	flags.AddTxFlagsToCmd(cmd)
+	return cmd
+}
+
+// NewMsgClawbackCmd returns a CLI command handler for creating a
+// MsgClawback transaction.
+func NewMsgClawbackCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "clawback [address]",
+		Short: "Transfer unvested amount out of a ClawbackVestingAccount.",
+		Long: `Must be requested by the original funder address (--from).
+		May provide a destination address (--dest), otherwise the coins return to the funder.
+		Delegated or undelegating staking tokens will be transferred in the delegated (undelegating) state.
+		The recipient is vulnerable to slashing, and must act to unbond the tokens if desired.`,
```

Comment:
```suggestion
		Long: `Must be requested by the original funder address (--from).
May provide a destination address (--dest), otherwise the coins return to the funder.
Delegated or undelegating staking tokens will be transferred in the delegated (undelegating) state.
The recipient is vulnerable to slashing, and must act to unbond the tokens if desired.
`,
```